apps/pie-docs/src/components/textarea/code/code.md

```diff
@@ -0,0 +1,192 @@
+---
+eleventyNavigation:
+    key: Code
+    parent: Textarea
+    order: 2
+shouldShowContents: true
+eleventyComputed:
+    props: "{% include './props.json' %}"
+    events: "{% include './events.json' %}"
+---
+
+## Overview
+
+<p>
+  <a href="https://www.npmjs.com/@justeattakeaway/pie-textarea" style="text-decoration: none">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-textarea.svg?label=pie-textarea">
+  </a>
+
+  <a href="https://www.npmjs.com/package/@justeattakeaway/pie-webc">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-webc.svg?label=pie-webc">
+  </a>
+</p>
+
+`pie-textarea` is a Web Component built with [Lit](https://lit.dev/), providing users with a larger input space for writing and editing text that requires more than one line.
+
+This component integrates easily with various frontend frameworks and can be customised through a set of properties.
+
+{% notification {
+  type: "information",
+  iconName: "engineers",
+  message: "You can try out this component on our [Storybook](https://webc.pie.design/?path=/docs/textarea) instance!"
+} %}
+
+## Installation
+
+To install `pie-textarea` in your application via `npm` or `yarn`:
+
+```shell
+npm i @justeattakeaway/pie-webc
+```
+
+```shell
+yarn add @justeattakeaway/pie-webc
+```
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "For more information on using PIE components as part of an application, check out the [Getting Started Guide.](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components)."
+} %}
+
+## Props
+
+{% componentDetailsTable {
+  tableData: props
+} %}
+
+## Events
+
+{% componentDetailsTable {
+  tableData: events
+} %}
+
+## Importing and usage in templates
+For HTML and Vue:
+
+```js
+// import as module into a js file that will be loaded on the page where the component is used.
+import '@justeattakeaway/pie-webc/components/textarea.js';
+```
+
+```html
+<pie-textarea
+    name="my-textarea"
+    placeholder="Please enter a value"
+    autocomplete="on"
+    label="Label"
+    maxLength="50"
+    value=""
+    autoFocus
+    readonly>
+</pie-textarea>
+```
+
+For React Applications:
+
+```jsx
+import { PieTextarea } from '@justeattakeaway/pie-webc/react/textarea.js';
+
+<PieTextarea
+    name="my-textarea"
+    placeholder="Please enter a value"
+    autocomplete="on"
+    label="Label"
+    maxLength="50"
+    value=""
+    autoFocus
+    readonly>
+</PieTextarea>
+```
+
+```jsx
+// React templates (using Next 13 and SSR)
+import { PieTextarea } from '@justeattakeaway/pie-textarea/dist/react';
```

Comment:
is it intended that we import `pie-webc` from React apps and individual packages for next?

Reply:
yes, it works for all components
for React templates (using Next 13 and SSR) we should import individual packages

Reply:
Hopefully it is clear but for Next 14, we do not import individual packages and still use the webc one :)